CMakeLists.txt

```diff
@@ -108,20 +118,13 @@ set(unused "${CMAKE_VERBOSE_MAKEFILE} ${CMAKE_RULE_MESSAGES}")
 # Set printer features
 set(PRINTERS_WITH_FILAMENT_SENSOR_YES "MINI")
 set(PRINTERS_WITH_GUI "MINI")
-set(WUI YES)
 
 if(${PRINTER} IN_LIST PRINTERS_WITH_FILAMENT_SENSOR_YES)
   set(FILAMENT_SENSOR YES)
 else()
   set(FILAMENT_SENSOR NO)
 endif()
 
-if(${PRINTER} IN_LIST PRINTERS_WITH_GUI)
-  set(GUI YES)
-else()
-  set(GUI NO)
-endif()
```

Comment:
This will break private.

Reply:
in private the build script has to be modified, disabling GUI if needed